__tests__/app.test.js

```diff
@@ -167,3 +171,89 @@ describe("task 6. GET /api/users", () => {
       });
   });
 });
+
+describe("task 7. get /api/arcticles/:arcticle_id", () => {
+  test("status:200, responds with updated articles table with extra property", () => {
+    const article_id = 1;
+    return request(app)
+      .get(`/api/articles/${article_id}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toEqual({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: "11",
+        });
+      });
+  });
+  test("7. status: 400, responds with invalid ", () => {
+    return request(app)
+      .get("/api/articles/WOWOIEIR")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid URL");
+      });
+  });
+
+  test("7. status: 404, responds with id error that doesnt exist ", () => {
+    return request(app)
+      .get("/api/articles/8989898")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("ID does not exist");
+      });
+  });
+});
+
+describe("task 8. get /api/arcticles", () => {
+  test("status:200, responds with updated articles table with user table info", () => {
+    return request(app)
+      .get(`/api/articles`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toEqual(12);
+      });
+  });
+  test("8.Make sure it responds in descending order", () => {
+    return request(app)
+      .get(`/api/articles`)
+      .expect(200)
+      .then(({ body }) => {
+        for (let i = 0; i < body.articles.length; i++) {
+          if (i + 1 === body.articles.length) {
+            return;
+          }
+          const thisDate = new Date(body.articles[i].created_at).valueOf();
+          const nextDate = new Date(body.articles[i + 1].created_at).valueOf();
+          if (thisDate < nextDate) {
+            throw new Error("failed not in descending order");
```

Comment:
we can use jest-sorted here which will tidy up this code and make what you're testing much more readable - i would strongly suggest refactoring this

Reply:
will refactor this now! then once both changes are made am I okay to merge?